github/installation.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
```

Comment:
I was merely trying to follow the structure of the other events, but that's a fairly special case, so maybe?